### PR TITLE
Add runbook for PrometheusRemoteWrite alert

### PR DESCRIPTION
**Pull request by DebakelOrakel**

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/monitoring/runbooks/prometheus_remotewrite.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+=== icon:[search] Investigate
+
+* Check that the remote write receiver's endpoint is reachable
++
+[source,bash]
+----
+kubectl -n openshift-monitoring --as=cluster-admin exec -it prometheus-k8s-0 -- curl <remote-write-endpoint>
+----
+* Check the remote write receiver for any issues.
+If the remote write receiver is a Mimir instance, check the Mimir nginx pod logs.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/monitoring/runbooks/prometheus_remotewrite.adoc:20]:
```suggestion
If the remote write receiver which has trouble is the VSHN central metrics Mimir instance, check the Mimir nginx pod logs on APPUiO Cloud cloudscale.ch LPG 2.
+
[source,bash]
----
kubectl -n vshn-appuio-mimir logs deploy/vshn-appuio-mimir-nginx --tail=200
----
+
```

File: docs/modules/ROOT/pages/how-tos/monitoring/runbooks/prometheus_remotewrite.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+== Alert Rule: SYN_PrometheusRemoteWriteBehind [[SYN_PrometheusRemoteWriteBehind]]
+== Alert Rule: SYN_PrometheusRemoteWriteDesiredShards [[SYN_PrometheusRemoteWriteDesiredShards]]
+
+=== icon:glasses[] Overview
+
+This alert may indicate that the remote write receiver isn't accepting metrics due to an internal problem or that there's a network issue between Prometheus and the remote write receiver.
+If the remote write receiver is a Mimir instance, the root cause may be that the ngnix in front of the Mimir components has stale pod IPs in its DNS cache.
+
+=== icon:[search] Investigate
```
Comment (haasad) [inline docs/modules/ROOT/pages/how-tos/monitoring/runbooks/prometheus_remotewrite.adoc:11]:
```suggestion
=== icon:search[] Investigate
```

**Review by simu — APPROVED** [review]
LGTM. Added some more details to check the VSHN central metrics Mimir

**Review by haasad — CHANGES_REQUESTED** [review]

**Review by haasad — APPROVED** [review]
LGTM, thanks for taking the time to document this :+1:

**Pull request merged**